tests/integration/security/authorization_test.go

```diff
@@ -392,23 +386,21 @@ func TestAuthorization_NegativeMatch(t *testing.T) {
 		Run(func(t framework.TestContext) {
 			ns := apps.Namespace1
 			ns2 := apps.Namespace2
-
+			dst0 := apps.VM.Match(echo.Namespace(apps.Namespace1.Name()))
```

Comment:
I think this has the similar reduced test coverage issue like I mentioned above, please see my other comment.

Reply:
I have reverted the tests back to b,c,d and add more for VM. I also add a TODO to revise the vm tests with multi-vm support and test updates.